frontend/public/components/image-stream.tsx

```diff
@@ -101,7 +101,7 @@ const ImageStreamTagsRow: React.SFC<ImageStreamTagsRowProps> = ({imageStream, sp
     </span>
     <span className="col-md-4 col-sm-4 hidden-xs co-break-all">
       {image && <React.Fragment>{image}</React.Fragment>}
-      {!image && '-'}
+      {!image && <React.Fragment><i className="pficon pficon-warning-triangle-o" aria-hidden="true" />&nbsp;There is no image associated with this tag.</React.Fragment>}
```

Comment:
I'm wondering if this should always be a warning. There are scenarios where it simply hasn't synced yet, correct?

Here is some of the logic we had in the past:

https://github.com/openshift/origin-web-console/blob/enterprise-3.5/app/views/browse/imagestream.html#L94-L105

Reply:
Re. the warning, if we can tell the difference btw a situation where there is some reason we can't get an image versus the image hasn't synced yet, then I am all for giving users the most correct information. Regardless, the IST shouldn't be a link if it will 404.